packages/validators/src/server/external/index.ts

```diff
@@ -24,7 +25,8 @@ export const CreateReviewSchema = z.object({
 
 export const EditReviewSchema = z.object({
   id: z.number().int().positive(),
-  restaurantId: z.number().int().positive(),
+  restaurantId: z.string(),
+  name: z.string(),
```

Comment:
```suggestion
  restaurantName: z.string(),
```
Try and restrict the value here as well (min and max lengths)

Reply:
done